product_docs/docs/biganimal/release/reference/cli.mdx

```diff
@@ -67,12 +78,12 @@ $ biganimal show-credentials
 ┃ ba-user1 ┃ portal.biganimal.com ┃ 443  ┃ Bearer           ┃ **********  ┃ **********   ┃ ********** ┃
```

Comment:
Has the messaging changed for show-credentials. It seems like we no longer display refresh and access tokens and only display the context credential. Can you confirm this change? If yes, I will update the output in the docs.
![image](https://user-images.githubusercontent.com/90263457/180402154-1e32097f-abf4-44fd-b730-093232e3dd61.png)

Reply:
Updated as communicated on Slack.